Faizrahmanov.Artur/lab1/main.cpp

```diff
@@ -0,0 +1,16 @@
+#include <iostream>
+#include <fstream>
+#include "FileMoverDaemon.h"
+
+int main(int argc, char** argv)
+{
+    if (argc != 2)
+    {
+        std::cout << "Incorrect arguments" << std::endl;
+        return EXIT_FAILURE;
+    }
+    FileMoverDaemon::getInstance()->initialize(argv[1]);
+    FileMoverDaemon::getInstance()->run();
+
+    return 0;
```

Comment:
Опционально:
Раз уж вверху используется EXIT_FAILURE, то можно
```suggestion
    return EXIT_SUCCESS;
```